src/command_modules/azure-cli-registry/azure/cli/command_modules/registry/_format.py

```diff
@@ -0,0 +1,107 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from collections import OrderedDict
+
+from ._utils import get_resource_group_by_registry
+
+def output_format(result):
+    '''Returns the list of container registries each of which is an ordered dictionary 
+    :param list/dict result: The container registry object(s) or deployment result(s)
+    '''
+    obj_list = result if isinstance(result, list) else [result]
+    return [_format(item) for item in obj_list]
+
+def _format(item):
+    '''Returns an ordered dictionary of the container registry or deployment result
+    :param dict item: The container registry object or deployment result
+    '''
+    if isinstance(item, dict) and 'id' in item and '/providers/Microsoft.Resources/deployments/' in item['id']:
+        return _format_deployment(item)
+    elif isinstance(item, dict) and 'id' in item and '/providers/Microsoft.Krater/registries/' in item['id']:
+        return _format_registry(item)
+    else:
+        raise ValueError('Unknown item: ' + str(item))
+
+def _format_deployment(item):
```

Comment:
I see that `_format_deployment(item)` has if statements testing the presence of properties and values in those properties. Are those properties and values conditional, or are we doing this to assert that they do indeed exist before accessing them?

If they are conditional, I'm okay with this code. If the code is only asserting the existence of these properties or their values, I'd like to figure out if there's a way to simply assert instead of having an if statement. For example:
```python
if 'resourceGroup' in item and 'item['resourceGroup']
```

would be something like
```python
assert 'resourceGroup in item
assert item['resourceGroup'
```

Let me know what you think. It may not be applicable to this method, I just found it funny to have so many if statements.